src/System.Private.CoreLib/src/System/Internal.cs

```diff
@@ -25,8 +25,10 @@
 #endif // FEATURE_COMINTEROP
 
 [assembly: DefaultDependencyAttribute(LoadHint.Always)]
```

Comment:
Nit: The DefaultDependencyAttribute and StringFreezingAttribute attributes can be just deleted here. They used to give hints to fragile NGen that we do not have anymore.